src/main/java/com/google/sps/EventMarker.java

```diff
@@ -0,0 +1,55 @@
+package com.google.sps;
+
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.model.EventDateTime;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date; 
+
+/**
+ * Marker data for each event to be added to the map.
+ */
+ public class EventMarker {
+    private String name;
+    private String description;
+    private String location; 
+    DateTime date; 
```

Comment:
these two should be private too. If they're being accessed by other classes, you can add getDate() and setDate() functions